compiler/damlc/tests/src/DA/Test/ShakeIdeClient.hs

```diff
@@ -1140,7 +1140,8 @@ scenarioTests mbScenarioService = Tasty.testGroup "Scenario tests"
           let vr = VRScenario foo "test"
           setFilesOfInterest [foo]
           setOpenVirtualResources [vr]
-          expectVirtualResourceRegex vr "Stack trace:.*- boom.*Foo:3:1.*- test.*Foo:5:1"
+          -- We dont get a stack trace anymore
+          expectVirtualResourceRegex vr "Aborted:  BOOM" --"Stack trace:.*- boom.*Foo:3:1.*- test.*Foo:5:1"
```

Comment:
Are you intending to merge it like this or is this just temporary to see how far CI gets? I find stack traces in scenarios to be quite useful.

Reply:
I agree regarding stack traces in scenarios and I'll make sure they stay when I do a detailed review of this PR.

Reply:
well.. I guess not.
But this was the only test failure I got in the ANF branch. And I spent a couple of fruitless hours this morning trying to figure out what had happened to the stack trace. But in the end I gave up and did what you see above.

The trouble with the stack traces is just they are so broken. And they get broken further by inlining and the work I've been doing on speedy. I just wonder if it might be better to completely remove them at the moment and implement them from the ground up.

And when we do that, we should have unit tests!
I find the shake tests so inscrutable. (for debugging at least)

Reply:
I going to debug why the stack trace gets lost in this example...

Reply:
this is fixed now